src/Microsoft.AspNet.JsonPatch/Operations/OperationOfT.cs

```diff
@@ -52,11 +52,5 @@ public void Apply([NotNull] TModel objectToApplyTo, [NotNull] IObjectAdapter ada
             }
```

Comment:
This class is not needed anymore right? Same case with `JsonPatchDocumentOfT`. Are you planning on removing it in one of the next PRs?